docs/source/CONTRIBUTING.md

```diff
@@ -5,7 +5,7 @@ OpenNMT-py is a community developed project and we love developer contributions.
 ## Guidelines
 Before sending a PR, please do this checklist first:
 
-- Please run `tools/pull_request_chk.sh` and fix any errors. When adding new functionality, also add tests to this script. Included checks:
+- Please run `onmt/tests/pull_request_chk.sh` and fix any errors. When adding new functionality, also add tests to this script. Included checks:
```

Comment:
Some things that we can do later though, I find the `pull_request_chk.sh` run the same cmd in `.travis.yml`, just wonder if we can call the script directly from `.travis.yml` rather than copy-paste.

Reply:
Yes, we can probably do this at the same time we'll look at the coverage CI stuff.